rdt/transformers/text.py

```diff
@@ -161,10 +163,10 @@ def _reverse_transform(self, data):
                     f"Please use a different regex for column ('{self.get_input_column()}')."
                 )
 
-            warnings.warn(
-                f"The data has {sample_size} rows but the regex for '{self.get_input_column()}' "
-                f'can only create {self.generator_size} unique values. Some values in '
-                f"'{self.get_input_column()}' may be repeated."
+            LOGGER.info(
+                "The data has %s rows but the regex for '%s' can only create %s unique values."
+                " Some values in '%s' may be repeated.",
+                sample_size, self.get_input_column(), self.generator_size, self.get_input_column()
             )
```

Comment:
We could do:
```python
message = (
    f"..."
    f"..."
)
``` 
But it is actually fine to have it as it is right now; The reason why, `f-string` gets 'compiled' always, meanwhile this way of string formatting gets generated only when called, which means that it will be generated only when `logger` is enabled.